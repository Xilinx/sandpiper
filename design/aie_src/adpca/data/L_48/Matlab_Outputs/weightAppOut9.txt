-1.521015167e-01 -1.731086254e+00
3.128206968e+00 -9.535599709e+00
-7.094202995e+00 -2.467684269e+00
