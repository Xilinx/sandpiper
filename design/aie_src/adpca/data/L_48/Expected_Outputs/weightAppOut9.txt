-1.521482468e-01 -1.731027722e+00 
3.128130674e+00 -9.535417557e+00 
-7.093951225e+00 -2.467679739e+00 
